app/src/main/java/org/oppia/android/app/topic/TopicFragmentPresenter.kt

```diff
@@ -136,25 +142,36 @@ class TopicFragmentPresenter @Inject constructor(
         classroomId,
         topicId,
         storyId,
-        enableExtraTopicTabsUi.value
+        enableTopicInfoTab.value,
+        enableTopicPracticeTab.value && enablePracticeTab
       )
     viewPager2.adapter = adapter
     TabLayoutMediator(tabLayout, viewPager2) { tab, position ->
-      val topicTab = TopicTab.getTabForPosition(position, enableExtraTopicTabsUi.value)
+      val topicTab = TopicTab.getTabForPosition(
+        position,
+        enableTopicInfoTab.value,
+        enableTopicPracticeTab.value && enablePracticeTab
+      )
```

Comment:
I would like you to review this old code, and determine whether we still need the code under `TabLayoutMediator`, since we have a function for returning tab position.